### ISSUE TITLE
Intercepting an Objective-C synthesized BOOL property setter causes a crash

### ISSUE THREAD

**Issue by JohnCoates** [frida/frida#344]
A crash occurs when intercepting the following property:
`@property(nonatomic, getter=isExample) BOOL example;`

Reproducing:
Run attached project on device, in release mode.
Run following commands:
```
frida -U -n FridaBoolHandlingBug
Interceptor.attach(new ApiResolver("objc").enumerateMatchesSync("-[ViewController setExample:]")[0].address, function (args) {} );
```
Tap on screen to call `setExample:`, crash ensues. Tested on iOS 9.0.2, iPhone 6+

[FridaBoolHandlingBug.zip](https://github.com/frida/frida/files/1315596/FridaBoolHandlingBug.zip)

**Comment by oleavr** [comment]
Fixed in frida/frida-gum@761aef463ee8b6a9cb7d932f59599e2401a0e808. Release coming soon.

Thanks a lot for your help! ❤️

**Comment by JohnCoates** [comment]
Awesome! Thanks for getting this fixed @oleavr 👏
